hotkey.js

```diff
@@ -14,10 +14,39 @@ function ignoreHotkeysIn(target) {
   )
 }
 
+function choseTheElement(current, selector) {
+  let list = current.querySelectorAll(selector)
+
+  for (let el of list) {
+    let closestElement = el.closest('data-keyux-ignore-hotkeys')
+    if (closestElement === current || !closestElement) {
+      return el
+    }
+  }
+  return null
+}
+
 function checkHotkey(where, code, overrides) {
   let codeOverride = overrides[code]
   if (Object.values(overrides).includes(code) && !codeOverride) return false
-  return where.querySelector(`[aria-keyshortcuts="${codeOverride || code}" i]`)
+  let current = where.activeElement
+  let cssSelector = `[aria-keyshortcuts="${codeOverride || code}" i]`
+  let elementWithHotKey = null
+
+  if (current.hasAttribute('data-keyux-hotkeys')) {
+    let id = current.getAttribute('data-keyux-hotkeys')
+    let newCurrent = where.getElementById(id)
+    elementWithHotKey = choseTheElement(newCurrent, cssSelector)
```

Comment:
If it returned `null` other ways to find elements will not be used.

We need something like:

```js
let click = findInside(where.byId(current.dataKeyuxHotkeys))
if (!click) {
  click = findInside(current)
  if (!lclick) {
    click = findInside(where)
  }
}
```